alamode.js

```diff
@@ -4,6 +4,9 @@
 var version = "0.23";
 
 var alamode = {
+  createAlamodeChartsWindowObj: (function() {
```

Comment:
I'm a little iffy on this but seems like the best to way to attache the object to the window without it creating a new object every time.

Reply:
It should only call this once, no matter how many time `addLinkToBigNumber` is called.